internal/provider/build.go

```diff
@@ -30,56 +31,56 @@ func fromImageData(data BuildResourceModel, wd string) (*build.Context, error) {
 	}
 	opts = append(opts,
 		build.WithImageConfiguration(ic),
-		// TODO(mattmoor): SBOMs would be nice
+		build.WithSBOMFormats([]string{"spdx"}),
```

Comment:
We _**could**_ make this configurable, but I'm not really sure where our CycloneDX support stands, and so I am biasing towards leaving this.  What do folks think?

Reply:
leave